respecConfig.json

```diff
@@ -3,14 +3,16 @@
   "group": "vc",
   "specStatus": "NOTE",
   "subtitle": "Verifiable Credentials Data Model v2.0 Interoperability Report",
-  "github": "https://github.com/w3c-ccg/vc2.0-test-suite",
-  "edDraftURI": "https://w3c-ccg.github.io/vc2.0-test-suite/#issue-credential",
+  "github": "https://github.com/digitalbazaar/vc-data-model-2-test-suite",
+  "edDraftURI": "https://www.w3.org/TR/vc-data-model-2.0/",
```

Comment:
The `edDraftURI` should be a link to the report, not the VCDM v2.0 data model. If you wanted to link to that spec, follow what we do in the "Related Specifications" section here:

https://github.com/w3c/vc-data-integrity/blob/main/index.html#L160-L175

... but only link to specifications that are related... which would be VCDM v2.0 (and that's it, for now).

Reply:
eddraftURI changes to the report url here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/31/commits/3b32c8ec8f71388cd94988049fdad360faef7c82

Reply:
VC 2.0 Data Model added to other links: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/31/commits/3af4ad8bd63ef93fcd37937f96bb3ad3358325e8